sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/service/schema/JsonSchemaManagerTest.java

```diff
@@ -0,0 +1,188 @@
+package io.stargate.sgv2.docsapi.service.schema;
```

Comment:
Is there a way to test the exact query that is returning from the observed result? Also what other cases should I test

Reply:
There are examples in the #1840 and #1844.. I test query strings in the dedicated query provider test and then check there here that the query executed on the bridge as same as the one that a query provider would return..